Makefile

```diff
@@ -7,7 +7,7 @@ UNAME := $(shell uname)
 
 PROTOC := protoc
  ifeq ($(UNAME), Darwin)
-	PROTOC = /opt/homebrew/opt/protobuf@27/bin/protoc
+	PROTOC = /opt/homebrew/opt/protobuf@21/bin/protoc
```

Comment:
Was showing that experimental issue which I though it was solved. So had to role back to the previous version we were using.